src/web/layouts/CommentLayout.tsx

```diff
@@ -187,10 +187,10 @@ export const CommentLayout = ({ CAPI, NAV }: Props) => {
 
     // Render the slot if one is true:
     // 1) The flag for this slot exists in the URL (i.e. ?slot-machine-flags=showBodyEnd)
-    // 2) The global switch for this slot is set to true;
+    // 2) The global switch for the Frontend/DCR Epic test is true
     const showBodyEndSlot =
         parse(CAPI.slotMachineFlags || '').showBodyEnd ||
-        CAPI.config.switches.slotBodyEnd;
```

Comment:
This `slotBodyEnd` was initially added to give us an easy way to turn on/off the rendering of this slot. The name was meant to be future proof and therefore unaware of the content eventually loaded into that Slot (currently an Epic, in the future anything else).
For simplicity, I've decided to not look at this switch while we run the DCR 5% test and instead implement one and only one switch that's specific for this test. In the future we might bring back slot specific switches but doesn't feel like we need them yet.